### ISSUE TITLE
Do you want assignments to be owned by many teachers?

### ISSUE THREAD

**Issue by arthurbernierjr** [Tstreets1000/Project-2-Gradebook#2]
https://github.com/Tstreets1000/Project-2-Gradebook/blob/c1c2b0d34f0e73a1d4c041b74dded9e9d2ebe35c/models/assignment.js#L6

**Comment by Tstreets1000** [comment]
No. I only want one or two assignments per teacher. Each teacher should have their own individual assignments.
